conf/modules/lidar_lite.xml

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="lidar_lite" dir="lidar">
```

Comment:
IMHO it makes sense to group the sensors based on their categories - i.e. lidars will be in `lidar` folder, sonars in `sonar` folder. If terraranger is a lidar then probably `lidars` (or `lidar`) folder would be logical?